boxoffice/views/admin_order.py

```diff
@@ -44,22 +44,22 @@ def jsonify_admin_orders(data_dict):
     item_collection_id = data_dict['item_collection'].id
     order_dicts = []
     for order in data_dict['orders']:
-        order_dicts.append({
-            'invoice_no': order.invoice_no,
-            'id': order.id,
-            'order_date': date_time_format(order.paid_at) if order.paid_at else date_time_format(order.initiated_at),
-            'status': order.is_confirmed,
-            'buyer_fullname': order.buyer_fullname,
-            'buyer_email': order.buyer_email,
-            'buyer_phone': order.buyer_phone,
-            'currency': CURRENCY_SYMBOL['INR'],
-            'amount': order.get_amounts().final_amount,
-            'url': '/ic/' + unicode(item_collection_id) + '/' + unicode(order.id),
-            'fully_assigned': order.is_fully_assigned() if order.is_confirmed else False,
-            'receipt': url_for('receipt', access_token=order.access_token),
-            'assignee': url_for('line_items', access_token=order.access_token),
-            'line_items': format_line_items(order.line_items)
-        })
+        if (order.is_confirmed):
+            order_dicts.append({
+                'invoice_no': order.invoice_no,
+                'id': order.id,
+                'order_date': date_time_format(order.paid_at) if order.paid_at else date_time_format(order.initiated_at),
```

Comment:
If you're checking for `order.is_confirmed`, `paid_at` won't be null. If it is, then it's a bug!